sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamComplexTypeTest.java

```diff
@@ -175,15 +178,14 @@ public void testRowWithArray() {
     pipeline.run().waitUntilFinish(Duration.standardMinutes(2));
   }
 
-  @Ignore("https://issues.apache.org/jira/browse/BEAM-5189")
```

Comment:
I don't see anything that changed in this test. Did this pass before?

Reply:
Nevermind, I see from your description you were using reserved keywords.

Reply:
The change at line 40 passes this test. 

In the past, `.addStringField("one")` uses Calcite reversed keyword `ONE`, which failed the test.